bin/checks-agent/src/env_provider.rs

```diff
@@ -54,7 +54,11 @@ impl ChecksAgentEnvProvider {
 
         let autodiscovery_provider = if in_standalone_mode {
             debug!("Using local autodiscovery provider due to standalone mode.");
-            BoxedAutodiscoveryProvider::from_provider(LocalAutodiscoveryProvider::new(vec!["dist"]))
+            let mut config_dir = config.get_typed_or_default::<String>("checks_config_dir");
+            if config_dir.is_empty() {
+                config_dir = "./dist/conf.d".to_string();
```

Comment:
This should be whatever the default Datadog Agent checks configuration directory is, like `/etc/datadog-agent/conf.d` or whatever.